fmtutils.c

```diff
@@ -39,7 +39,6 @@
 
 #include "ftmacros.h"
 
-#include <stddef.h>
```

Comment:
On Windows, this file uses `wchar_t`, so it has a reason of its own to include stddef.h.

Reply:
...unless we make an explicit commitment that pcap-int.h will always include stddef.h.

Reply:
So long as `pcap-int.h` continues to use `offsetof()`, it will include `<stddef.h>`. Arguably, each file that needs `offsetof()` ought to do the same instead of relying on another header. Let me see how it fits.

Reply:
stddef.h is for more than `offsetof()`; it's also for `ptrdiff_t`, `size_t`, `wchar_t`, and `NULL`, some of which are also defined in other headers.  Files that need one of those should include *some* header that defines it.